db/mainphotos copy.sql

```diff
@@ -0,0 +1,74 @@
+CREATE SCHEMA IF NOT EXISTS mainphotos;
+
+CREATE TABLE mainphotos.neighborhoods (
```

Comment:
Your naming convention works but I think with pqsl you can just call the table neighborhoods without the period. Same with the rest of the tables